commands/user.bee.inc

```diff
@@ -98,6 +187,180 @@ function users_bee_callback($arguments, $options) {
   );
 }
 
+/**
+ * Command callback: Create a user.
+ */
+function user_bee_create_callback($arguments, $options) {
+  $user = user_load_by_name($arguments['username']);
+  if ($user) {
+    bee_message(bt("User account '!user' allready exits.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+  if(empty($arguments['mail'])){
+    bee_message(bt("Mail argument missing!", array()), 'error');
+    return;
+  }
+  $check = db_select('users', 'u')
+    ->fields('u', array('uid'))
+    ->condition('mail', $arguments['mail'], '=')
+    ->execute()
+    ->fetchField();
+
+  if($check != '') {
+    bee_message(bt("A user account with mail '!mail' allready exits.", array(
+      '!mail' => $arguments['mail'],
+    )), 'error');
+    return;
+  }
+  $password = !empty($arguments['password']) ? $arguments['password'] : user_password(20);
+  $new_user = array(
+    'name' => $arguments['username'],
+    'pass' => $password,
+    'mail' => $arguments['mail'],
+    'status' => 1,
+    'access' => '0',
+  );
+  if ($account = entity_create('user', $new_user)) {
+    $account->save();
+    bee_message(bt("User '!user' has been created, with password !pass", array(
+      '!user' => $arguments['username'], '!pass' => $password,
+    )), 'success');
+  }else {
+    bee_message(bt("Could not create a new user account with the name, !user", array(
+      '!user' => $name,
+    )), 'error');
+  }
+}
+
+/**
+ * Command callback: Delete/cancel a given user.
+ */
+function user_cancel_bee_callback($arguments, $options) {
+  // Load the user account.
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' cound not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+
+  user_cancel(array(
+    'user_cancel_notify' => TRUE,
+    'user_cancel_method' => 'user_cancel_delete',
```

Comment:
I appreciate that drush had no options for this, but I wonder if we should expose the notify option and cancel method. I'll leave it up to you as they could always be added later if there is a need.